CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.8.0
+
+### Vision
+- Start monitoring performance related device info
+- Performance improved
```

Comment:
think we can get rid of `- Performance improved`. There's no any payload

Reply:
done